### PR TITLE
Upgrade apollo boost and add websockets

### PR DESCRIPTION
**Pull request by erezmus**
Closes libero/reviewer#673

### CODE REVIEW

File: src/core/utils/createApolloClient.ts [hunk 1]
```diff
@@ -39,6 +43,43 @@ export default (host: string): ApolloClient<unknown> => {
                 clearToken();
                 window.location.reload();
             }
+        }
+
+        if (networkError) {
+            console.log(`[Network error]: ${networkError}`);
```
Comment (Cooryd) [inline src/core/utils/createApolloClient.ts]: should this be a `console.error` to make it stand out when debuging? (I know the Apollo docs say console.log but this seems like something we might want to catch with newrelic)?

Comment (diversemix) [reply src/core/utils/createApolloClient.ts]: Ditto for line 35

Comment (norisalsaadie) [reply src/core/utils/createApolloClient.ts]: I believe this was existing code prior to the PR. While it's a valid concern it should probably become it's own ticket as it's out of scope of this PR

File: jest.config.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
     },
     globals: {
         CONFIG: {
-            API_HOST: '',
+            API_HOST: 'http://localhost',
```
Comment (norisalsaadie) [inline jest.config.js]: Should this have a port?!

File: src/core/utils/createApolloClient.ts (hunk 1 above)
Comment (Cooryd) [reply src/core/utils/createApolloClient.ts]: https://github.com/libero/reviewer/issues/674

File: jest.config.js (hunk 2 above)
Comment (erezmus) [reply jest.config.js]: this is just for the tests, so not really necessary

**Pull request merged**